### PR TITLE
Improve document about Github Actions section

### PR DESCRIPTION
**Pull request by jeffreytse**
This PR improves the section of GitHub actions for Jekyll users.

### CODE REVIEW

**Review by mattr- — DISMISSED** [review]
Thanks for taking the time and effort to update these docs! I've left a few suggestions I'd like you to address when you're able to.

**Comment by jeffreytse** [comment]
Hi @mattr- 

Firstly, hope you have a nice weekend. Are there any other issues on this PR? Or if you think it's not a good improvement, I will close it. : )

Thanks & Regards

File: docs/_docs/continuous-integration/github-actions.md [hunk 1]
```diff
@@ -197,10 +221,12 @@ successful deploy from the Action.
 - [jekyll-actions-quickstart] is an unofficial repository that includes a live demo of the
   `jekyll-actions` action. That project can be used as a template for making a new site.
 - [jekyll-action-ts] is another action to build and publish Jekyll sites on GiHub Pages that includes HTML formatting options with Prettier and caching.
+- [jekyll-deploy-action] is a Github Action to deploy the Jekyll site conveniently for GitHub Pages (An alternative action with better speed and compatibility).
```
Comment (mattr-) [inline docs/_docs/continuous-integration/github-actions.md:224]:
```suggestion
- [jekyll-deploy-action] is a GitHub Action to deploy the Jekyll site conveniently for GitHub Pages (An alternative action with better speed and compatibility).
```
One small tweak here for consistency.

**Review by jeffreytse — COMMENTED** [review]
Thank you! Now the small tweak is ready. Hope you have a good day! : )

**Review by mattr- — APPROVED** [review]

**Comment by mattr-** [comment]
@jekyllbot: merge +docs

**Pull request merged**